__tests__/integration/core-blockchain/blockchain.test.ts

```diff
@@ -262,8 +262,8 @@ describe("Blockchain", () => {
 
             expect(unconfirmedTransactions.transactions.length).toBe(transactionsWithoutType2.length);
 
-            expect(unconfirmedTransactions.transactions).toEqual(
-                transactionsWithoutType2.map(transaction => transaction.serialized.toString("hex")),
+            expect(unconfirmedTransactions.transactions.sort()).toEqual(
```

Comment:
You can use [.toIncludeAllMembers](https://github.com/jest-community/jest-extended#toincludeallmembersmembers) then you don't need to sort.